client/src/components/ChooseGenres.js

```diff
@@ -54,13 +56,42 @@ function ChooseGenres() {
     return <span>Error: {error.message}</span>;
   }
 
+  const handleClick = (event) => {
+    event.preventDefault();
+
+    const newSelectedGenres = [];
+    if (selectGenres.indexOf(event.target.innerHTML) === -1) {
+      selectGenres.push(event.target.innerHTML);
```

Comment:
I think it would be better to use `{genre}` instead of `innerHTML` here. I could just change the `innerHTML` and then the wrong value would be pushed :)